### ISSUE TITLE
in iOS Text coming from SpeechRecognizer disappears in TextBox when attempting to edit

### ISSUE THREAD

**Issue by stezelMIT** [mit-cml/appinventor-sources#3282]
**Describe the bug**
In iOS Text coming from SpeechRecognizer disappears in TextBox when attempting to edit.  It is colored grayish which looks like perhaps it is pasted as a Hint initially that disappears when the user clicks on the TextBox to edit the Text.
This request is coming from our CS teacher Kamran at DHA in Dubai.

**Affects**

- [ ] Designer
- [ ] Blocks editor
- [ ] Projects Explorer
- [ ] Companion
- [ ] Compiled apps
- [ ] Buildserver
- [ ] Debugging
- [ X] Other... (please describe)
Affects the GUI created

**Expected behavior**
in Android there is no issue trying to edit an SpeechRecognizer generated Text in a TextBox.
Not sure why so in iOS.

**Steps to reproduce**
Press button, speak, and once the text is pasted, try to edit it.
[speech_recogniser.aia.zip](https://github.com/user-attachments/files/18067174/speech_recogniser.aia.zip)

**Comment by stezelMIT** [comment]
[speech_recogniser.aia.zip](https://github.com/user-attachments/files/18067188/speech_recogniser.aia.zip)

The file is this.

**Comment by ewpatton** [comment]
This seems to only affect the textbox when Multiline is set to True, in this project via the Designer. If Multiline isn't required, then setting that to false (the default) should be sufficient for now.

**Comment by stezelMIT** [comment]
I believe the project needs Multiline option checked.
It is only an iOS issue right?
________________________________

**Comment by ewpatton** [comment]
> It is only an iOS issue right?

Yes. I've submitted a PR and it is being processed for deployment via TestFlight.

**Comment by stezelMIT** [comment]
I tested and it is working for me.
Much appreciated Evan!
